src/validation/fields/helper/github-handles.validation.helper.ts

```diff
@@ -0,0 +1,18 @@
+import { ErrorMessages } from '../../error.messages';
+
+export const validateGithubHandles = (value: string) => {
+    if (!value) {
+        throw new Error(ErrorMessages.GIT_HANDLE);
+    }
+
+    const handles = value.split(',').map(handle => handle.trim());
```

Comment:
Add test case for submitting handles with spaces either side of the `value` string?